jujugui/static/gui/src/app/components/account/credentials/credentials.js

```diff
@@ -348,19 +349,23 @@ YUI.add('account-credentials', function() {
       let addButton = (
         <juju.components.GenericButton
           action={this._toggleAdd}
-          type="inline-base"
-          title="Add"
+          type="inline-neutral"
+          title={this.state.showAdd ? 'Cancel' : 'Add'}
```

Comment:
It does. It basically toggles the 'add credentials' expendingRow... I don't particularly like the way this work, but I figure it's an improvement.